ols/src/llms/providers/provider.py

```diff
@@ -89,6 +92,32 @@ class ProviderParameter:
     PROVIDER_WATSONX: WatsonxParameters,
 }
 
+# Generic to Azure OpenAI parameters mapping
+AzureOpenAIParametersMapping: dict[str, str] = {}
+
+# Generic to OpenAI parameters mapping
+OpenAIParametersMapping: dict[str, str] = {}
+
+# Generic to BAM parameters mapping
+BAMParametersMapping: dict[str, str] = {}
+
+# Generic to Watsonx parameters mapping
+WatsonxParametersMapping: dict[str, str] = {
+    "min_new_tokens": GenParams.MIN_NEW_TOKENS,
+    "max_new_tokens": GenParams.MAX_NEW_TOKENS,
+    "top_k": GenParams.TOP_K,
+    "top_p": GenParams.TOP_P,
+    "temperature": GenParams.TEMPERATURE,
```

Comment:
if these are to be our official "generic llm parameter names", should we make them into constants?

Reply:
yup